docs/howto/partitions.rst

```diff
@@ -94,20 +106,34 @@ Defined partitions may be referenced in the ``organize``, ``stage``, and
   prime:
     - (<partition>)/<path>
 
-Paths in the configuration yaml not beginning with a partition label will
-implicitly use the default partition.
+Paths in the project file not beginning with a partition label will implicitly
+use the default partition.
 
 The source path of an ``organize`` entry can only be from the default
-partition.  For example:
+partition.  For example, this is valid:
+
+.. code-block:: yaml
+
+  organize:
+    usr/local/bin/hello: (component/bar-baz)/bin/hello
```

Comment:
it's a bit confusing that this uses `components`, which as far as I know is only mentioned in passing as an example of partition use

Reply:
Why is that confusing?  Isn't `components` essentially just another namespace?

If need be, I can rewrite the example to use something that would make sense going into the `kernel` partition.

Reply:
Yes, but even partition namespaces are only briefly mentioned in this document. I think this would be clearer if first you have an example without a namespace, and then one with it. 
Or at least add a comment on this yaml, something like `# organize usr/local/bin/hello into the "bar-baz" partition in the "component" namespace`

Reply:
Added explanatory text: https://github.com/canonical/craft-application/pull/371/commits/5d05a6e37945939ff0951bc8b956df4503853ea5